### ISSUE TITLE
I try first with my gmail account and it says Wrong password (all its ok) i dont know why it says that if you can help me please. Thanks

### ISSUE THREAD

**Issue by g0dsito** [d4az/hack-gmail#13]

**Comment by ebadfd** [comment]
yes sure please email me with your issue I will get back to you dasithsv@gmail.com

**Comment by bokunodev** [comment]
your library `(rockyou.txt)` is not a valid list of string.
it contains random binary in it.
also you should strip any white spaces and convert it to utf-8 without BOM.
\-
also, you should use imap or pop3 client, instead of smtp.
android gmail app uses imap.

**Comment by ebadfd** [comment]
huh really ? ill fix it and push a new one
